src/routes.json

```
@@ -72,6 +72,14 @@
         "view": "conference/2018/schedule/schedule",
         "title": "Conference Schedule"
     },
+    {
+        "name": "conference-index-2019",
+        "pattern": "^/conference/2019/?$",
+        "routeAlias": "/conference(?!/201[4-5])",
```

I've forgotten why we need this - Ray would remember. It would be good to refresh everyone's knowledge.

---

The routeAlias is just a way for us to reduce the size of a giant regex in Fastly. But I think at this point we don't have to worry about that size limit? We should check. In case it still matters, the regex in routeAlias should match the pattern for the route, so this should change like so:
```suggestion
        "routeAlias": "/conference(?!/201[4-9])",
```

And the point of the alias is to group many similar patterns together, so anything that was using the `/conference(?!/201[4-5])` routeAlias should be updated to the new alias.

---

Done